giskard/scanner/llm/llm_basic_sycophancy_detector.py

```diff
@@ -34,7 +36,7 @@ def run(self, model: BaseModel, dataset: Dataset) -> Sequence[Issue]:
             return [
                 Issue(
                     model,
-                    dataset1,
+                    dataset2,
```

Comment:
Yes, we use it to generate the tests. This is a workaround to enable debugging later because usually dataset2 contains more relevant examples. It‘s not super reliable, but a quick fix.